common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -563,14 +563,20 @@ def definition_to_xml(self, resource_fs):
                 if key in self.fields and self.fields[key].is_set_on(self):
                     try:
                         xml.set(key, unicode(value))
-                    except ValueError as exception:
-                        exception_message = "{message}, Block-location:{location}, Key:{key}, Value:{value}".format(
-                            message=exception.message,
+                    except (ValueError, UnicodeDecodeError) as exception:
+                        exception_message = "Block-location:{location}, Key:{key}, Value:{value}".format(
                             location=unicode(self.location),
                             key=key,
-                            value=unicode(value)
+                            value=value
                         )
-                        raise ValueError(exception_message)
+                        log.exception(exception_message)
+
+                        # If exception is UnicodeDecodeError then set value using unicode 'utf-8' scheme.
+                        if type(exception) == UnicodeDecodeError:
+                            log.info("Setting xml value using 'utf-8' scheme.")
```

Comment:
Can we that it like
```
except UnicodeDecodeError as ex:
# Code for UnicodeDecodeError
except ValueError as ex:
# Code for ValueError
```

Just to avoid `if` condition in the except.